server/superdesk/commands/clean_images.py

```diff
@@ -19,27 +19,27 @@ class CleanImages(superdesk.Command):
     This command will remove all the images from the system which are not referenced by content.
     Probably running db.repairDatabase() is needed in Mongo to shring the DB size.
     """
+
     def run(self):
         try:
             print('Starting image cleaning.')
             used_images = set()
 
-            archive_items = superdesk.get_resource_service('archive').get(None, {'type': 'picture'})
+            archive_items = superdesk.get_resource_service('archive').get_from_mongo(None, {'type': 'picture'})
```

Comment:
Actually is to get from storage server and not from search server. Probably we just have to rename it.